aspnetcore/toc.md

```diff
@@ -29,6 +29,7 @@
 ## [Open Web Interface for .NET (OWIN)](fundamentals/owin.md)
 ## [Choose between ASP.NET Core and ASP.NET](choose-aspnet-framework.md)
 ## [Choose between .NET Core and .NET Framework runtime](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server)
+## [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage)
```

Comment:
Dan makes a good point, but as long as you did make it an article you might still consider moving it up.  An alternative is to convert this into the runtime store doc he's talking about, i.e. cover both in one and make runtime store the primary topic.  That might be closer to what Dan had in mind.